c++11/src/logger.cpp

```diff
@@ -0,0 +1,11 @@
+#include "logger.h"
+
+namespace lightstep {
+//------------------------------------------------------------------------------
+// GetLogger
+//------------------------------------------------------------------------------
+spdlog::logger& GetLogger() {
+  static auto logger = spdlog::stderr_logger_mt("lightstep");
```

Comment:
I worry just slightly about the size vs. benefit of the spdlog library, given that we're using it to report relatively rare things to the console, I'd be happy using `<iostream>` APIs for this just as well. 

Elsewhere in this PR review I've suggested that we provide a way for users to send these messages somewhere other than stderr, that applies to this. I'm not asking you to remove spdlog at this time--but I'm on the fence about it. Do you know what API the lyft/envoy codebase uses for logging?